func/internal/podevaluator.go

```diff
@@ -648,7 +648,10 @@ func (pm *podManager) getImageMetadata(ctx context.Context, ref name.Reference,
 }
 
 func getImage(ctx context.Context, ref name.Reference, auth authn.Authenticator, image string, enablePrivateRegistries bool, enablePrivateRegistriesTls bool, tlsSecretPath string) (containerregistry.Image, error) {
-	if enablePrivateRegistries && !strings.HasPrefix(image, defaultRegistry) && enablePrivateRegistriesTls {
+	// if private registries or their appropriate tls configuration are disabled in the config we pull image with default operation otherwise try and use their tls cert's
+	if !enablePrivateRegistries || strings.HasPrefix(image, defaultRegistry) || !enablePrivateRegistriesTls {
+		return remote.Image(ref, remote.WithAuth(auth), remote.WithContext(ctx))
+	} else {
```

Comment:
you don't need the `else`, if you delete it you can move the indentation of the block below to the left

Reply:
but if i remove the else will the TLS setup in the else not just fire for any scenario?
a scenario being for example (enablePrivateRegistries=true , strings.HasPrefix(image, defaultRegistry)=false and enablePrivateRegistriesTls=false) in this example i don't want the content in the else to fire but it will if i remove it no? or am i mistaken?
the content in that else should only ever fire if (enablePrivateRegistries=true , strings.HasPrefix(image, defaultRegistry)=false and enablePrivateRegistriesTls=true) in all other cases it should just do what is in the if statement.